tests/accounts.test.js

```diff
@@ -0,0 +1,760 @@
+                response.should.have.status(200);
+                response.body.should.be.a("array");
+                response.body.length.should.be.eq(1);
+                done();
+            });
+        });
+
+        it("GET /Accounts Name Not Found", (done) =>{
+            chai.request(server).get("/accounts/name/Luisa")
+            .auth(accessToken, { type: 'bearer' })
+            .end( (error, response) => {
+                response.should.have.status(404);
+                response.body.should.have.property("message");
+                response.body.should.have.property("message").eq("No se encontro registro(s)");
+                done();
+            });
+        });
+
+        it("GET /Accounts Name Bad Request", (done) =>{
+            chai.request(server).get("/accounts/name/12352")
+            .auth(accessToken, { type: 'bearer' })
+            .end( (error, response) => {
+                response.should.have.status(400);
+                response.body.should.have.property("message");
+                done();
+            });
+        });
+    });
+    
+    describe("GET Tests Accounts AGE", () => {
+        it("GET /Accounts Age", (done) =>{
+            chai.request(server).get("/accounts/age/21")
+            .auth(accessToken, { type: 'bearer' })
+            .end( (error, response) => {
+                response.should.have.status(200);
+                response.body.should.be.a("array");
+                response.body.length.should.be.eq(2);
+                done();
+            });
+        });
+
+        it("GET /Accounts Age Not Found", (done) =>{
+            chai.request(server).get("/accounts/age/14")
+            .auth(accessToken, { type: 'bearer' })
+            .end( (error, response) => {
+                response.should.have.status(404);
+                response.body.should.have.property("message");
+                response.body.should.have.property("message").eq("No se encontro registro(s)");
+                done();
+            });
+        });
+
+        it("GET /Accounts Age Bad Request", (done) =>{
+            chai.request(server).get("/accounts/age/Martha")
+            .auth(accessToken, { type: 'bearer' })
+            .end( (error, response) => {
+                response.should.have.status(400);
+                response.body.should.have.property("message");
+                done();
+            });
+        });
+    });
+
+    describe("GET Tests Accounts DateBirth", () => {
+        it("GET /Accounts DateBirth", (done) =>{
+            chai.request(server).get("/accounts/dateBirth/2000-07-05")
+            .auth(accessToken, { type: 'bearer' })
+            .end( (error, response) => {
+                response.should.have.status(200);
+                response.body.should.be.a("array");
+                response.body.length.should.be.eq(1);
+                done();
+            });
+        });
+
+        it("GET /Accounts DateBirth Not Found", (done) =>{
```

Comment:
Codacy has a fix for the issue: [Missing space after =>.](https://app.codacy.com/gh/Martha-Miroslava/APIBiziidiDiidxaza/pullRequest?prid=8267260)
```suggestion
        it("GET /Accounts DateBirth Not Found", (done) => {
```